sdk/tests/deqp/functional/gles3/es3fNegativeTextureApiTests.js

```diff
@@ -1147,9 +1147,11 @@ goog.scope(function() {
             gl.texImage2D(gl.TEXTURE_2D, 0, gl.R8I, 0, 0, 0, gl.RED_INTEGER, gl.BYTE, null);
             gl.generateMipmap(gl.TEXTURE_2D);
             this.expectError(gl.INVALID_OPERATION);
-            gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA32F, 0, 0, 0, gl.RGBA, gl.FLOAT, null);
-            gl.generateMipmap(gl.TEXTURE_2D);
-            this.expectError(gl.INVALID_OPERATION);
+            if (!gl.getExtension('EXT_color_buffer_float') || !gl.getExtension('OES_texture_float_linear')) {
```

Comment:
That's correct. The original code should have always generated an INVALID_OPERATION.

A new clause here could read something like:

if (gl.getExtension('EXT_color_buffer_float') && gl.getExtension('OES_texture_float_linear')) {
  gl.generateMipmap(gl.TEXTURE_2D);
  this.expectError(gl.NO_ERROR);
}